assets/src/dashboard/app/components/logo.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function WebStoriesLogo() {
+  return (
+    <svg
```

Comment:
Can we move this to a svg file? We use some infra to make it available to us as a React component.

Reply:
That was my original intent but the Wordpress server only appears to serve the js file and the style css. I have been working with some of the team to determine how to serve assets beyond those two with little success.

Reply:
Our build should pack them in JS. Essentially it's no different from what you did - just done automatically for you.

Reply:
Perfect. In the past we have used a babel plugin to do this. Any objections to that?

Reply:
That's what we do. See [this example](https://github.com/google/web-stories-wp/blob/192d430bb7ffc2dd17fedf191136736640da8770/assets/src/edit-story/components/button/index.js#L26).